### PR TITLE
Allowing to specify alternative language versions of the page

### PR DESCRIPTION
**Pull request by humanbeinc**
https://support.google.com/webmasters/answer/2620865

### CODE REVIEW

File: src/SimpleMvcSitemap.Tests/XmlAssertionExtensions.cs [hunk 1]
```diff
@@ -12,7 +14,10 @@ public static void BeXmlEquivalent(this StringAssertions assertions, [PathRefere
             XmlDocument doc = new XmlDocument { PreserveWhitespace = false };
             doc.Load(filename);
 
-            assertions.Subject.Should().Be(doc.InnerXml);
+            XDocument doc1 = XDocument.Parse(File.ReadAllText(filename));
```
Comment (uhaciogullari) [inline src/SimpleMvcSitemap.Tests/XmlAssertionExtensions.cs]: Why did you need this change?

**Comment by uhaciogullari** [comment]
There seems to be an error with pushing results to Coverall. I will fix that. Do you need this merged urgently?

**Comment by humanbeinc** [comment]
@uhaciogullari No, I can use my fork.

File: src/SimpleMvcSitemap.Tests/XmlAssertionExtensions.cs (hunk 1 above)
Comment (humanbeinc) [inline src/SimpleMvcSitemap.Tests/XmlAssertionExtensions.cs]: To make a node-by-node integral comparison rather than a string comparision

**Comment by uhaciogullari** [comment]
@humanbeinc Thank you very much for the PR. I did a renaming in a few places. New version is also on [NuGet](https://www.nuget.org/packages/SimpleMvcSitemap/2.4.0)
